Makefile

```diff
@@ -20,11 +17,9 @@ endif
 
 # For release builds we link statically libstdc++ and libgcc. Currently,
 # all the release builds are performed by gcc.
-ifeq ($(HELIO_RELEASE),y)
-	LINKER_FLAGS += -static-libstdc++ -static-libgcc
-endif
+LINKER_FLAGS += -static-libstdc++ -static-libgcc
 
-HELIO_FLAGS = $(if $(HELIO_RELEASE),-release $(HELIO_RELEASE_FLAGS),) \
+HELIO_FLAGS = -DHELIO_RELEASE_FLAGS="-g" \
 			  -DCMAKE_EXE_LINKER_FLAGS="$(LINKER_FLAGS)" \
```

Comment:
You could fix the odd indentation while here :)

Reply:
not sure what you mean